Packs/CommonScripts/Scripts/ArrayToCSV/ArrayToCSV.py

```diff
@@ -0,0 +1,24 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+from traceback import format_exc
+
+
+def arr_to_csv_command(array: list[str] | str) -> CommandResults:
+    csv = ','.join(array)
+    return CommandResults(
+        readable_output=csv
+    )
+
+
+def main():  # pragma: no cover
```

Comment:
Why add Pragma no cover?
Please remove

Reply:
I need it in order to raise the unit tests coverage from 50% should I remove it and count on the force merge?

Reply:
Nope, you should try to add more coverage to the script if possible

Reply:
impossible. i have 100% coverage of the code until the main.